Source/Internal/IGListAdapterInternal.h

```diff
@@ -17,6 +17,11 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+// Generate the string representation of a reusable view class when registering with a UICollectionView
+NS_INLINE NSString *IGListReusableViewIdentifierForClass(Class viewClass, NSString * _Nullable nibName, NSString * _Nullable kind) {
```

Comment:
very minor nits: 😄 

- use 3 `///` for comment (for xcode docs)
- docs: "Generates a string..." and `.` at the end 
- naming: `IGListReusableViewIdentifier()`